site/it/tutorials/keras/regression.ipynb

```diff
@@ -0,0 +1,900 @@
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "sD7qHCmNIOY0"
+      },
+      "outputs": [],
+      "source": [
+        "EPOCHS = 1000\n",
+        "\n",
+        "history = model.fit(\n",
+        "  normed_train_data, train_labels,\n",
+        "  epochs=EPOCHS, validation_split = 0.2, verbose=0,\n",
+        "  callbacks=[tfdocs.modeling.EpochDots()])"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "tQm3pc0FYPQB"
+      },
+      "source": [
+        "Visualizziamo il progresso del modello durante l'addestramento usando le statistiche immagazzinate nell'oggetto `history`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "4Xj91b-dymEy"
+      },
+      "outputs": [],
+      "source": [
+        "hist = pd.DataFrame(history.history)\n",
+        "hist['epoch'] = history.epoch\n",
+        "hist.tail()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "czYtZS9A6D-X"
+      },
+      "outputs": [],
+      "source": [
+        "plotter = tfdocs.plots.HistoryPlotter(smoothing_std=2)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "nMCWKskbUTvG"
+      },
+      "outputs": [],
+      "source": [
+        "plotter.plot({'Basic': history}, metric = \"mae\")\n",
+        "plt.ylim([0, 10])\n",
+        "plt.ylabel('MAE [MPG]')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "N9u74b1tXMd9"
+      },
+      "outputs": [],
+      "source": [
+        "plotter.plot({'Basic': history}, metric = \"mse\")\n",
+        "plt.ylim([0, 20])\n",
+        "plt.ylabel('MSE [MPG^2]')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "AqsuANc11FYv"
+      },
+      "source": [
+        "Questo grafico mostra miglioramenti modesti, o anche peggioramenti nell'errore di validazione dopo circa 100 epoche. Andiamo a modificare la chiamata alla `model.fit` per fermare automaticamente l'addestramento quando il risultato della validazione non migliora. Useremo la *EarlyStopping callback* che controlla la condizione di addestramento ad ogni epoca. Se un dato numero di epoche passano senza mostrare miglioramenti, allora l'addestramento viene interrotto automaticamente.\n",
+        "\n",
+        "[Qui](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/EarlyStopping) Potete imparare di più a proposito di questa callback."
```

Comment:
Here there should be a small typo in the uppercase 'P' of 'Potete' which should be lower case since it's not at the beginning of the sentence